### PR TITLE
Add ONLYOFFICE DesktopEditors 5.3.5

### LINKED ISSUE #2742: [Request] Add ONLYOFFICE Desktop Editors
Hi,

There is not yet a 32-bit portable edition. There is no way to check the current version of this .zip file, even from their downloads website, there is no public portable edition. I'll try to have the authors to provide them in a more consistent way.

onlyoffice.json

```json
{
    "version": "5.3.95.506",
    "license": "AGPLv3",
    "url": "http://repo-doc-onlyoffice-com.s3-eu-west-1.amazonaws.com/windows/onlyoffice-desktopeditors/latest/DesktopEditors_x64.zip",
    "hash": "md5:d51fba54a99ada0f8cbbe44a5cdde261",
    "homepage": "https://www.onlyoffice.com/apps.aspx",
    "bin": "DesktopEditors.exe",
    "shortcuts": [
        [
            "DesktopEditors.exe",
            "ONLYOFFICE Desktop Editors 5.3"
        ]
    ]
}
```

### PR DESCRIPTION
**Pull request by gothicx**
Closes #2742

### CODE REVIEW

File: bucket/onlyoffice.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/ONLYOFFICE/DesktopEditors/releases/download/ONLYOFFICE-DesktopEditors-$version/DesktopEditors_x64.exe#/dl.7z"
+            },
+            "32bit": {
+                "url": "https://github.com/ONLYOFFICE/DesktopEditors/releases/download/ONLYOFFICE-DesktopEditors-$version/DesktopEditors_x64.exe#/dl.7z"
+            }
+        }
+    }
+}
```
Comment (linsui) [inline bucket/onlyoffice.json]:
```suggestion
}

```

File: bucket/onlyoffice.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        },
+        "32bit": {
+            "url": "https://github.com/ONLYOFFICE/DesktopEditors/releases/download/ONLYOFFICE-DesktopEditors-5.3.5/DesktopEditors_x86.exe#/dl.7z",
+            "hash": "sha256:546199e2bf5bb7880ff2762819525f591d0c82626a4fc1729cfd3ca746415565"
+        }
+    },
+    "shortcuts": [
+        [
+            "DesktopEditors.exe",
+            "ONLYOFFICE Desktop Editors 5.3"
```
Comment (linsui) [inline bucket/onlyoffice.json]:
```suggestion
            "ONLYOFFICE Desktop Editors"
```

File: bucket/onlyoffice.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+﻿{
+    "version": "5.3.5",
+    "license": "AGPLv3",
+    "homepage": "https://www.onlyoffice.com/apps.aspx",
+    "innosetup": true,
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/ONLYOFFICE/DesktopEditors/releases/download/ONLYOFFICE-DesktopEditors-5.3.5/DesktopEditors_x64.exe#/dl.7z",
```
Comment (linsui) [inline bucket/onlyoffice.json]:
```suggestion
            "url": "https://github.com/ONLYOFFICE/DesktopEditors/releases/download/ONLYOFFICE-DesktopEditors-5.3.5/DesktopEditors_x64.exe",
```

If it's an innosetup installer, keep exe.

File: bucket/onlyoffice.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            "hash": "sha256:356002deff774aa4bec1b382b3ac7495a34ee8087e02d29d7e1be73091051b96"
```
Comment (linsui) [inline bucket/onlyoffice.json]:
```suggestion
            "hash": "356002deff774aa4bec1b382b3ac7495a34ee8087e02d29d7e1be73091051b96"
```

File: bucket/onlyoffice.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    },
+    "shortcuts": [
+        [
+            "DesktopEditors.exe",
+            "ONLYOFFICE Desktop Editors 5.3"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/ONLYOFFICE/DesktopEditors",
+        "regex": "\/releases\/tag\/\\w+-\\w+-([\\d\\.]+)"
```
Comment (linsui) [inline bucket/onlyoffice.json]:
```suggestion
        "regex": "/releases/tag/[\\w-]+([\\d.]+)"
```

File: bucket/onlyoffice.json (hunk 4 above)
Comment (linsui) [reply bucket/onlyoffice.json]: You can checkver and autoupdate by `.\bin\checkver.ps1 onlyoffice -u -f`

**Comment by linsui** [comment]
You can just use @Ash258 's manifest, it's better 👍

File: bucket/onlyoffice.json [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/ONLYOFFICE/DesktopEditors/releases/download/ONLYOFFICE-DesktopEditors-$version/DesktopEditors_x64.exe"
+            },
+            "32bit": {
+                "url": "https://github.com/ONLYOFFICE/DesktopEditors/releases/download/ONLYOFFICE-DesktopEditors-$version/DesktopEditors_x86.exe"
+            }
+        }
+    }
+}
```
Comment (linsui) [inline bucket/onlyoffice.json]:
```suggestion
}

```

File: bucket/onlyoffice-desktopeditors.json [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/ONLYOFFICE/DesktopEditors/releases/download/ONLYOFFICE-DesktopEditors-$version/DesktopEditors_x64.exe"
+            },
+            "32bit": {
+                "url": "https://github.com/ONLYOFFICE/DesktopEditors/releases/download/ONLYOFFICE-DesktopEditors-$version/DesktopEditors_x86.exe"
+            }
+        }
+    }
+}
```
Comment (linsui) [inline bucket/onlyoffice-desktopeditors.json]:
```suggestion
}

```

**Comment by Ash258** [comment]
12 changed files?? What are you doing?

**Comment by gothicx** [comment]
Moving to a branch.

**Comment by gothicx** [comment]
I did this wrong and did the change in the fork master and anything I do there, it will trigger a change here, so I moved it to a branch as I should have done initially.

**Pull request closed without merging**